vars/buildPlugin.groovy

```diff
@@ -187,8 +187,8 @@ def call(Map params = [:]) {
     }
 }
 
-static boolean hasDockerLabel() {
-    env?.NODE_LABELS?.contains("docker")
+boolean hasDockerLabel() {
+    env.NODE_LABELS?.contains("docker")
```

Comment:
`env.NODE_LABELS` is never null, either, which is why I proposed dropping the second `?` in my suggestion. Harmless of course.

Reply:
https://issues.jenkins-ci.org/browse/INFRA-2210 FTR